src/connectors/Fortmatic.ts

```diff
@@ -26,20 +26,18 @@ export class FortmaticConnector extends FortmaticConnectorCore {
 
     const provider = this.fortmatic.getProvider()
 
-    const pollForOverlayReady = new Promise<void>((resolve) => {
-      const interval = setInterval(() => {
-        if (provider.overlayReady) {
-          clearInterval(interval)
-          this.emit(OVERLAY_READY)
-          resolve()
-        }
-      }, 200)
-    })
-
-    const [account] = await Promise.all([
-      provider.enable().then((accounts: string[]) => accounts[0]),
-      pollForOverlayReady,
-    ])
+    // TODO implements this
+    // const pollForOverlayReady = new Promise<void>((resolve) => {
+    //   const interval = setInterval(() => {
+    //     if (provider.overlayReady) {
+    //       clearInterval(interval)
+    //       this.emit(OVERLAY_READY)
+    //       resolve()
+    //     }
+    //   }, 200)
+    // })
```

Comment:
What was this for? Do you intend to enable it again or is it no needed for the new version?

Reply:
That's the point, I checked to see if it is necessary. I did not find that it was necessary after much testing. What I did find is that it is responsible for the strange behavior was commented where the modal reappears:
![Selection_454](https://user-images.githubusercontent.com/4270166/153506216-759dd85b-3f75-4813-9120-a4a364d47239.png)


As it is not part of the `custom` I have not removed it, let me know if you think I should remove it.

Reply:
Ah, good point, I missed it's the original source file!

Then you should mod it instead and do the same thing (comment it out without deleting it)